src/executor/executor.ts

```diff
@@ -387,6 +399,19 @@ export class TaskExecutor {
   }
 
   /**
+   * @deprecated This method is marked for removal in a future release.
+   * Migrate your code by using `Array.map` and `Promise.all` instead.
```

Comment:
I think the "Migrate" part has to also be on the same line to get properly display in IDE - minor serverity,

Reply:
In VSCode you always get the same message `'X' is deprecated` so here it makes no difference. Not sure about other IDEs

![image](https://github.com/golemfactory/golem-js/assets/34130112/7893f5ff-4a32-4aa5-9b27-afe63fcdcf24)